### ISSUE TITLE
Elements with missing roles

### ISSUE THREAD

**Issue by juliushaertl** [nextcloud/deck#3822]
- [x] Sidebar button in top right is missing an accessible name
- [x] Expand button in the app navigation is missing a name
	- [ ] should use aria-expanded

**Comment by marcelklehr** [comment]
The expand button is fixed in latest nc vue.

**Comment by JuliaKirschenheuter** [comment]
Is this issue still open?

**Comment by marcelklehr** [comment]
Technically yes as the expand button in Nextcloud/vue doesn't have an aria-expanded when I last checked

**Comment by juliushaertl** [comment]
It has been added with https://github.com/nextcloud/nextcloud-vue/commit/eb4735ea03219fef2f8cb60d4b365f26662f5395 and I just verified that it is there also with the latest deck release.
